### ISSUE TITLE
Add List.Nonempty.last

### ISSUE THREAD

**Issue by MartinSStewart** [mgold/elm-nonempty-list#12]
It would be nice to have the following function since it's guaranteed to not return Maybe
```elm
{-| Return the last element of the list.
-}
last : Nonempty.Nonempty a -> a
last nonempty =
    Nonempty.tail nonempty |> List.Extra.last |> Maybe.withDefault (Nonempty.head nonempty)
```
The `List.Extra.last` function can be found [here](https://package.elm-lang.org/packages/elm-community/list-extra/latest/List-Extra#last). Probably best to copy the source code to avoid adding a dependency to `elm-community/list-extra`.

**Comment by mgold** [comment]
Released 4.1.0.

I don't regret adding it, but you could have also used `get -1`.

**Comment by MartinSStewart** [comment]
Oh, good point. I guess the disadvantage of `get -1` is that it isn't as discoverable. I completely forgot you could do that.
